src/components/scenes/SendScene2.tsx

```diff
@@ -531,6 +532,8 @@ const SendComponent = (props: Props) => {
   const renderSelectFioAddress = () => {
     if (hiddenFeaturesMap.fioAddressSelect) return null
     const fioTarget = spendInfo.spendTargets.some(target => target.otherParams?.fioAddress != null)
+    if (!fioTarget) return null
```

Comment:
This seems safe but only due to distributed logic that we currently always send a fioTarget if we send a fioRequest although they are not technically both needed. What motivated this change?

Reply:
Not sure... It may have been needed for some prior version of my changes, it doesn't seem
necessary anymore. Removing.